File: frontend/src/app/dto/ReportScore.ts

export class ReportScore {

  veryWell: number;
  well: number;
  moderate: number;
  badly: number;
  veryBadly: number;
  validAnswers: number;
  totalAnswers: number;
  mean: string;
  medianstring: string;
  standardDeviation: string;
  yourMean: string;
  yourMedian: string;
  ebbiScore: string;
  ebbiIndex: string;

  constructor(veryWell, well, moderate, badly, veryBadly, mean, medianstring, standardDeviation, yourMean, yourMedian, ebbiScore, ebbiIndex) {
    this.veryWell = veryWell;
    this.well = well;
    this.moderate = moderate;
    this.badly = badly;
    this.veryBadly = veryBadly;
    this.mean = medianstring;
    this.standardDeviation = standardDeviation;
    this.yourMean = yourMean;
    this.yourMedian = yourMedian;
    this.ebbiScore = ebbiScore;
    this.ebbiIndex = ebbiIndex;
  }

}

File: src/main/java/is/larsen/ebbi/Model/ReportScore.java
package is.larsen.ebbi.Model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ReportScore {
    private Integer veryWell;
    private Integer well;
    private Integer moderate;
    private Integer badly;
    private Integer veryBadly;
    private Integer validAnswers;
    private Integer totalAnswers;
    private String mean;
    private String median;
    private String standardDeviation;
    private String yourMean;
    private String yourMedian;
    private String ebbiScore;
    private String ebbiIndex;

}
